java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -236,18 +237,14 @@ public MantaClient(final ConfigContext config) {
     }
 
     /**
-     * Creates a new instance of the Manta client with the specified connection factory.
-     * This method is typically used by unit tests to inject connection factory
-     * customizations so we can simulate different network events.
+     * Creates a new instance of the Manta client based on user-provided connection objects. This allows for a larger
+     * degree of customization at the cost of
```

Comment:
Trailing off?

Reply:
Must've been interrupted, thanks for catching that.